src/app.rs

```diff
@@ -74,15 +72,7 @@ impl LatestPatchsetsState {
     }
 
     pub fn fetch_current_page(self: &mut Self) -> color_eyre::Result<()> {
-        if let Err(failed_feed_request) = self
-            .lore_session.process_n_representative_patches(&self.lore_api_client, self.page_size * &self.page_number) {
-            match failed_feed_request {
-                FailedFeedRequest::UnknownError(error) => bail!("[FailedFeedRequest::UnknownError]\n*\tFailed to request feed\n*\t{error:#?}"),
-                FailedFeedRequest::StatusNotOk(feed_response) => bail!("[FailedFeedRequest::StatusNotOk]\n*\tRequest returned with non-OK status\n*\t{feed_response:#?}"),
-                FailedFeedRequest::EndOfFeed => (),
-            }
```

Comment:
So in this case, if the request returns an `Err(EndOfFeed)`, this caller should return an Ok(()) rather than propagating the error

My fault, didn't read the code with enough attention